package.json

```diff
@@ -14,7 +14,7 @@
     "posttest": "npm run docker:stop",
     "test:unit": "nyc ava test/unit",
     "test:integration": "docker-compose run test && npm run migrate:local && nyc ava test/integration; npm run docker:stop",
-    "test:docker:ci": "npm run migrate:local && ava test/unit test/integration/*-test.js --verbose",
+    "test:docker:ci": "npm run migrate:local && ava test/unit/*/*-test.js test/integration/*/*-test.js --verbose",
```

Comment:
@austinkelleher Is this a constraint we want to impose? (That all test files will be exactly three levels deep? i.e. test/<unit|integration>/*-test.js)